src/routes/docs/products/auth/preferences/+page.markdoc

```diff
@@ -0,0 +1,407 @@
+---
+layout: article
+title: Preferences
+description: Learn how to store and manage user preferences in Appwrite. Compare storage options and pick the right approach for your application.
+---
+
+Preferences allow users to customize their experience in your application. You can save settings like theme choice, language selection, or notification preferences. Appwrite provides multiple ways to store these preferences, depending on your needs.
+
+There are four main options for storing user preferences in applications using Appwrite:
+
+# Browser localStorage {% #localstorage %}
+
+The browser's localStorage API is a standard web technology that persists data in the user's browser.
+
+- Device-specific: Settings are only available on the current device
+- Simple key-value storage
+- No server-side processing required
+- Data persists even after browser sessions end
+- Limited to 5MB per origin in most browsers
+
+```javascript
+// Store a preference using the browser's built-in localStorage API
+localStorage.setItem('darkMode', 'true');
+
+// Retrieve a preference
+const darkMode = localStorage.getItem('darkMode');
+```
+
+# User preferences {% #user-preferences %}
+
+Appwrite provides a built-in user preferences system through the Account API, allowing you to store preferences directly on the user object.
+
+- Persists across all user devices
+- Stored as a JSON object
+- Limited to 64kB of data
+- Automatically synced when the user logs in
```

Comment:
What do you mean by this? where is it synced to?

Reply:
That makes sense. I've removed the syncing line